src/components/SearchBar/style.css

```diff
@@ -0,0 +1,13 @@
+.search-bar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+input[type="text"] {
```

Comment:
Can keep your HTML tags/ body, * selectors on top above CSS. Keeping CSS specificity in mind. I know you would have done it here unknowingly, but keep that in mind while writing vanilla CSS.